net/strongswan/files/ipsec.init

```diff
@@ -334,9 +398,14 @@ config_ipsec() {
 }
 
 prepare_env() {
+	local seen_setup=
```

Comment:
Don't think you need the =.  In fact, if you're unlucky, or didn't notice, local is actually a function, and has a return code, so these sorts of things are quite different.  (not relevant here, but just something to ponder with local...

```
local output=$(something that might fail)
[ $? -eq 0 ] && {
    printf "hoho, this will always run, because local will 'never' fail\n"
}
```
vs
```
local output
output=$(something that might fail)
[ $? -eq 0 ] && {
    printf "this will do what you expected...\n"
}
```

Reply:
> Don't think you need the =. In fact, if you're unlucky, or didn't notice, local is actually a function, and has a return code, so these sorts of things are quite different. (not relevant here, but just something to ponder with local...

I figured this was the same as:

```
local seen_setup=""
```

I suppose I could also set it to `0` and then test for `[ $seen_setup -eq 1 ] && return` ... any preferences?

Reply:
Ended up explicitly setting it to `""`.